convert-llama-h5-to-gguf.py

```diff
@@ -68,18 +69,23 @@ def permute(weights: NDArray, n_head: int) -> NDArray:
 
 print("gguf: get model metadata")
 
-llm_arch    = "llama"
-head_count  = hparams["num_attention_heads"]
+llm_arch = "llama"
+hf_repo = hparams["_name_or_path"]
+head_count = hparams["num_attention_heads"]
+head_count_kv = hparams["num_key_value_heads"]
 block_count = hparams["num_hidden_layers"]
 
 gguf_writer.add_name(last_dir)
 gguf_writer.add_architecture(llm_arch)
+gguf_writer.add_quantization_version(ftype)
```

Comment:
Quantization version is only used if any tensors are quantized. I guess a description of ftype should be stored to `general.file_type: string`

Reply:
ftype 0: "All tensors F32"
ftype 1: "Most tensors F16, some F32"

Reply:
They're all in the same enumeration in llama.cpp anyway and reported when loading the model. It's replaced by the target quantization scheme in `gguf_file_saver` during quantization.

Reply:
gguf_writer.add_file_type( "All tensors F32" if ftype == 0 else "Most tensors F16, some F32")

Reply:
quantization version is for backwards compatibility when reusing the same quant identifiers / enum like Q4_0 
current quantization version is 2 for all quantization formats

Reply:
https://github.com/ggerganov/ggml/pull/302#discussion_r1285164498

Reply:
https://github.com/ggerganov/llama.cpp/blob/1d60468eeec4bbe7dfde704297877fa23072081c/ggml.h#L202-L203

Reply:
Removed.